tests/e2e-leg-3/non-cluster-admin-deployment/15-run-cluster-admin-tasks.yaml

```diff
@@ -0,0 +1,48 @@
+# (c) Copyright [2021-2023] Micro Focus or one of its affiliates.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Setup the namespace for the operator
+  - script: kubectl apply -n $NAMESPACE -f ../../../config/release-manifests/operator-rbac.yaml
+  # Install the CRDs
+  - command: sh -c "cd ../../.. && make install"
+---
+# Give user 'fred' general access to edit things inside its namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: fred-cluster-access
```

Comment:
Wasn't the goal to give limited access to the cluster.

Reply:
User Fred needs some access to do things like create pods, etc. This is a RoleBinding so it only gives Fred access to to do things inside the namespace.